content/docs/state-and-lifecycle.md

```diff
@@ -193,19 +189,19 @@ ReactDOM.render(
 );
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/KgQpJd?editors=0010)
+[**Փորձել CodePen-ում**](https://codepen.io/gaearon/pen/KgQpJd?editors=0010)
 
-Next, we'll make the `Clock` set up its own timer and update itself every second.
+Հաջորդիվ, `Clock`-ում մենք կտեղադրենք իր սեփական ժամաչափը, որը կթարմացնի ինքն իրեն ամեն վայրկյան։
```

Comment:
էլի set up կա

Reply:
"we'll make the `Clock` set up "-ը բառացի "մենք այնպես կանենք, որ/կստիպենք, որ Clock-ը գործի դնի/մեկնարկի իր սեփական ժամաչափը"